common/names.go

```diff
@@ -4,12 +4,12 @@ import (
 	"fmt"
 	"os"
 
-	localv1 "github.com/openshift/local-storage-operator/pkg/apis/local/v1"
+	localv1 "github.com/openshift/local-storage-operator/api/v1"
 	corev1 "k8s.io/api/core/v1"
 )
 
 const (
-	defaultDiskMakerImageVersion = "quay.io/openshift/origin-local-storage-diskmaker"
+	defaultDiskMakerImageVersion = "quay.io/pjiandan/local-diskmaker"
```

Comment:
we should fix these before having final PR?

Reply:
fixed